oak_protobuf_idl_tests/oak.protobuf_idl.tests.rs.txt

```diff
@@ -38,32 +37,29 @@ impl <S: TestService> ::oak_protobuf_idl::Transport for TestServiceServer<S> {
 
 impl <S: TestService> TestServiceServer<S> {
     fn invoke_inner(&mut self, request_bytes: &[u8]) -> Result<::prost::alloc::vec::Vec<u8>, ::oak_protobuf_idl::Status> {
-        use ::prost::Message;
         let request = ::oak_protobuf_idl::Request::decode(request_bytes).map_err(|err| {
             ::oak_protobuf_idl::Status::new_with_message(
                 ::oak_protobuf_idl::StatusCode::Internal,
-                format!("Client failed to deserialize the response: {:?}", err),
+                ::oak_protobuf_idl::format!("Client failed to deserialize the response: {:?}", err),
```

Comment:
Why is this a `.txt` file? And is there an instruction on how to update it?

Reply:
yes, it's currently explained in the generate.rs file already. But I can add a README.